dissect/target/plugin.py

```diff
@@ -553,6 +553,7 @@ def _walk(osfilter: str = None, root: dict = None) -> Iterator[PluginDescriptor]
 
     if (
         osfilter
+        and not hasattr(osfilter, "__is_default__")
```

Comment:
```suggestion
        and not osfilter is DefaultPlugin
```

Maybe also fix the typing of osfilter, as `type[OSPlugin]` it is a OSPlugin subclass.

Reply:
This won't work due to cyclic imports.

Reply:
Ah nasty

Reply:
Would it still be an idea to refactor the typing of `plugins(osfilter: str ...` to `plugins(osfilter: type[OSPlugin] ...`?

Reply:
Yes, that would be better, working on that...

Reply:
Forgot there are some Python tricks for this. Implemented.